app-backend/batch/src/main/scala/com/azavea/rf/batch/airflow/export/CreateExportDef.scala

```diff
@@ -0,0 +1,165 @@
+package com.azavea.rf.batch.airflow.export
```

Comment:
Okay I'm with you now; I don't have a strong opinion either way, but as you mentioned I put the `[...]/airflow/*` setup in place to essentially split the "out-facing" interface that the airflow task uses from the core Spark-related stuff.